Dockerfile

```diff
@@ -10,8 +10,9 @@ RUN apt-get update && apt-get install -y \
 COPY . /home/node/app
 
 RUN cd /home/node/app && npm install 
+RUN yarn global add nodemon
 
 WORKDIR /home/node/app
 EXPOSE 3000
 
-CMD node app.js
+CMD nodemon -L app.js
```

Comment:
this should be something like `npm run XXX` with an related script in package.json